doc/dev/installation.rst

```diff
@@ -414,3 +414,123 @@ To generate html files for the documentation for Catalyst:
   make docs
 
 The generated files are located in ``doc/_build/html``
+
+Install a Frontend-Only Development Environment from TestPyPI Wheels
+--------------------------------------------------------------------
+
+It is possible to work on the source code repository and test the changes without 
+having to compile Catalyst. This is ideal for situations where the changes do not target the 
+runtime or the MLIR infrastructure, and only concern to the frontend. It basically 
+makes use of the shared libraries already shipped with the TestPyPI Catalyst wheels.
+
+Essential Steps
+^^^^^^^^^^^^^^^
+
+To activate the development environment, open a terminal and issue the following commands:
+
+.. code-block:: console
+
+  # Clone the Catalyst repository  
+  git clone --recurse-submodules --shallow-submodules https://github.com/PennyLaneAI/catalyst.git
+
+  # Activate the development environment based on the latest TestPyPI wheels (please provide a name)
+  cd catalyst
+  . ./activate_dev_from_wheel.sh myenv
+
+To exit the virtual environment, type:
+
+.. code-block:: console
+
+  deactivate
+
+How Does it Work?
+^^^^^^^^^^^^^^^^^
```

Comment:
This is nice, but personally I find this section a little too detailed to appear in the docs. If you want to describe the principle in one or two sentences I think that would be more appropriate for the audience we are targeting.

Reply:
Done